### ISSUE TITLE
Code has something different points with paper

### ISSUE THREAD

**Issue by csg1718** [visiontao/ncd#1]
Hello, I was interested in your result on pgm extrapolation and trained model with your code.
But I got 14.303% accuracy which is quite worse performance than your result.
The differences are my experiment settings, so i changed batch size with 32 and workers with 4.
However i dont think it made that large gap.
So I read your paper one more time and noticed that dropout is not implemented in this github code.

Could I get the code which made 24.9% accuracy on pgm extrapolation?

**Comment by csg1718** [comment]
> Hello, I was interested in your result on pgm extrapolation and trained model with your code. But I got 14.303% accuracy which is quite worse performance than your result. The differences are my experiment settings, so i changed batch size with 32 and workers with 4. However i dont think it made that large gap. So I read your paper one more time and noticed that dropout is not implemented in this github code.
> 
> Could I get the code which made 24.9% accuracy on pgm extrapolation?

I'm sorry. there was it.
Could I get the pretrained model which made 24.9% accuracy on pgm extrapolation?
